.vscode/launch.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Not sure, I dont think I was able to get it working actually, I should just remove it